BB-SC/MWLayer/Platform-Health-Management/BB_Watchdog.md

```diff
@@ -0,0 +1,77 @@
+
+# Watchdog
+
+## BB Tags(s)
+<!-- Tag(s) define in which area(s) (cloud, in-vehicle) the BB is executed, and what type of BB it is (tool, process, microservice) -->
+BB-SC
+
+## Functional Clusters
+<!-- In which Functional Cluster the BB be located; if none of the existing fit new required -->
+Platform Health Management
+
+## Layer
+<!-- AppLayer, MWLayer, OSLayer, HWLayer -->
+MWLayer
```

Comment:
Isn't this an OSLayer concern?

Reply:
described as "Extend the traditional concept of watchdog concept ", thus probably ok in MW Layer

Reply:
consider renaming/giving a more precise name

Reply:
is there a proposal for a more precise name?